rules-tests/Php82/Rector/Class_/ReadOnlyClassRector/Fixture/skip_already_readonly.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Rector\Tests\Php82\Rector\Class_\ReadOnlyClassRector\Fixture;
+
+use MyCLabs\Enum\Enum;
+
+readonly class SkipAlreadyReadonly
```

Comment:
@TomasVotruba phpunit seems got error when scan already readonly class run on php 8.1

Note: PR on php-src is not yet merged too https://github.com/php/php-src/pull/7305 for php 8.2 feature.

```
There was 1 error:

1) Rector\Tests\Php82\Rector\Class_\ReadOnlyClassRector\ReadOnlyClassRectorTest::test with data set #0 (Symplify\SmartFileSystem\SmartFileInfo Object (...))
PHPStan\Parser\ParserErrorsException: Syntax error, unexpected T_READONLY on line 7

phar:///Users/samsonasik/www/rector-src/vendor/phpstan/phpstan/phpstan.phar/src/Parser/RichParser.php:54
```

@ondrejmirtes any way to tweak phpstan config to make it work?

Reply:
@samsonasik This will need to be added in https://github.com/nikic/PHP-Parser first

Reply:
ok, I will remove this already fixture for now, transformation itself seems working ok.

```diff
-class A
+readonly class A
```